Swagger/PlayFab/CloudScript.swagger.json

```diff
@@ -98,7 +94,11 @@
           "description": "Detailed description of individual issues with the request object",
           "type": "object"
         }
-      }
+      },
+      "required": [
```

Comment:
Hmmm... is the "required" list meaningful for you?
It's a response, not a request, so required shouldn't actually have any relevance/meaning.

Reply:
Yes, it affects codegen. If it's not required, AutoRest will generate nullable types.